toolkits/nature/packages/nature-header/HISTORY.md

```diff
@@ -1,5 +1,8 @@
 # History
 
+## 5.2.0 (2021-03-17)
```

Comment:
Not just a patch?

Reply:
Good point - probably should be - I'll update it